Develop

DamsYaYa opened the pull request:
Проект седьмого спринта

DamsYaYa committed:
Обновлены библиотеки
Добавлены тесты, намечена структура

DamsYaYa committed:
Добавлены импорты для allure

DamsYaYa committed:
Добавлены импорты для allure

alesyaisachenko commented on src/main/java/Constant/Constant.java:
```diff
@@ -0,0 +1,8 @@
+package Constant;
```
Это нужно исправить:

package Constant -- нужно package constant, пакеты именуются в таком формате.

Подправь, плиз, этот момент во всем коде.

alesyaisachenko commented on src/main/java/Steps/CourierSteps.java:
```diff
@@ -0,0 +1,53 @@
+    }
+
+    @Step("Удаление созданного курьера")
+    public static void deleteCourier() {
```
Можно лучше:

лучше передавать id Coirier, которого нужно удалить

alesyaisachenko commented on src/test/java/GetListOfOrdersTest.java:
```diff
@@ -0,0 +1,29 @@
+
+    @Test
+    @DisplayName("Получение списка заказов")
+    public void GetOrdersTest() {
```
Это нужно исправить:

 GetOrdersTest() -- нужно  getOrdersTest(), методы именуются в таком формате

alesyaisachenko replied:
эта ошибка не исправлена, ее нужно исправить

alesyaisachenko commented on src/test/java/GetListOfOrdersTest.java:
```diff
@@ -0,0 +1,29 @@
+    @Test
+    @DisplayName("Получение списка заказов")
+    public void GetOrdersTest() {
+        ValidatableResponse response = order.getOrders();
```
Это нужно исправить:

тест падает на этом шаге -- все тесты должны проходить

alesyaisachenko replied:
эта ошибка также не исправлена, ее нужно исправить

alesyaisachenko commented on src/test/java/LoginFailedTest.java:
```diff
@@ -0,0 +1,52 @@
+import static org.apache.http.HttpStatus.SC_NOT_FOUND;
+
+@DisplayName("Негативные тесты на логин")
+public class LoginFailedTest {
```
Это нужно исправить:

в этом классе все тесты падают -- все тесты должны проходить

alesyaisachenko commented on src/test/java/LoginTest.java:
```diff
@@ -0,0 +1,38 @@
+
+        ValidatableResponse response = courierSteps.loginWithCourier(Courier.getRandomCourier());
+
+        response.assertThat().body("id", greaterThan(0)).and().statusCode(SC_CREATED);
```
Это нужно исправить:

этот тест также падает (статус код нужно другой)

DamsYaYa committed:
Переименованы пакеты
Пофикшено падение теста на получение списка заказов
Удалены импорты junit5
Мелкие улучшения

DamsYaYa committed:
Пофикшены тесты на логин с невалидными данными

alesyaisachenko commented on src/test/java/GetListOfOrdersTest.java:
```diff
@@ -0,0 +1,29 @@
+import io.restassured.RestAssured;
+import io.restassured.response.ValidatableResponse;
+import org.junit.Before;
+import org.junit.jupiter.api.DisplayName;
```
Это нужно исправить:

в pom-файле нет настройки на юпитер.

Сейчас тесты не компилируются из-за неправильного импорта

DamsYaYa replied:
этот тест полностью переписан, в следующем файле GetOrderLis все замечания были учтены

alesyaisachenko commented on src/test/java/LoginFailedTest.java:
```diff
@@ -0,0 +1,86 @@
+        RestAssured.baseURI= "https://qa-scooter.praktikum-services.ru";
+        courier = Courier.getRandomCourier();
+        courierSteps = new CourierSteps();
+        courierSteps.createCourier(courier);
```
Это нужно исправить:

после прохождения тестов нужно удалять созданного курьера

DamsYaYa committed:
Delete GetListOfOrdersTest.java

удален файл с неактуальным кодом теста получения списка заказов. тест переписан и находится в файле GetOrderListTest

DamsYaYa committed:
Добавлен метод удаления созданного курьера

DamsYaYa committed:
Merge remote-tracking branch 'origin/develop' into develop
